code/modules/vehicles/segway.dm

```diff
@@ -5,6 +5,11 @@
 	icon_state = "secway"
 	callme = "secway"
 	anchored = 0 //This is not so heavy
+	move_speed = 12
```

Comment:
That was a bit of experimentation with attempting to make it slower which I forgot to remove. Dunno if it makes this a DNM PR though as it didn't seem to affect anything.